### ISSUE TITLE
error in dotnet husky install

### ISSUE THREAD

**Issue by EngRajabi** [alirezanet/Husky.Net#89]
I don't have this problem in version 0.5.4. That's why I use the old version
### Version

0.6.0

### Details

add command in root sln directory
```csharp
dotnet husky install
```
error
```csharp
failed to execute command 'git'
Could not determine git configuration
```

### Steps to reproduce

- dotnet new tool-manifest - success
- dotnet tool install Husky - success
- dotnet husky install - error

**Comment by alirezanet** [comment]
@Tychus Can you check this error if you have time, I quickly checked and it is related to Submodule check that you've added recently.

**Comment by alirezanet** [comment]
Hi @EngRajabi,

I tested it and it looks fine! at least with the steps you mentioned to reproduce the error:

- mkdir test
- cd test
- git init
- dotnet new tool-manifest 
- dotnet tool install Husky 
- dotnet husky install

what did I miss?

**Comment by EngRajabi** [comment]
For testing, I did this process with version 0.5.4 and everything was fine. But with version 0.6.0 I still have problem.
Indra.sln file exists in root.
```csharp
D:\Indra Project>dotnet husky install -v
Checking git path and working directory
failed to execute command 'git'
Command execution failed, see the inner exception for details.

Standard error:
fatal: cannot change to 'D:\Indra': No such file or directory
Could not determine git configuration
```

**Comment by Tychus** [comment]
I can take a look. from reading the issue i suspect it's around the running `git` in a specific directory.

**Comment by Tychus** [comment]
Ah wait a second your path has space in it.  That might be the issue i think. 

`D:\Indra Project>dotnet husky install -v`

error says `fatal: cannot change to 'D:\Indra': No such file or directory`

**Comment by alirezanet** [comment]
Good catch, I think the problem is related to these methods where we're sending the path to the git command:
https://github.com/alirezanet/Husky.Net/blob/6a64ebed307797f37c8e36c1c4624f7875a88eb5/src/Husky/Services/Git.cs#L111-L115

the `ExecBufferedAsync` should receive a list of arguments instead of string

**Comment by Tychus** [comment]
working on a fix for those two methods

**Comment by alirezanet** [comment]
Hi @EngRajabi,
I released a new version and this should be fixed in `v0.6.1`, please let me know if it works for you.

**Comment by EngRajabi** [comment]
Yes, the problem is solved. Thankful
